src/constants/dexToZapMapping.ts

```diff
@@ -0,0 +1,26 @@
+import { ChainId } from '@ape.swap/sdk'
+import { LiquidityDex, ZapVersion } from '../types'
+
+export const dexToZapMapping: Record<LiquidityDex, Partial<Record<ChainId, ZapVersion>>> = {
+  [LiquidityDex.ApeSwapV2]: {
+    [ChainId.BSC]: ZapVersion.ZapV1,
+    [ChainId.BSC_TESTNET]: ZapVersion.ZapV1,
+    [ChainId.MATIC]: ZapVersion.ZapV1,
+    [ChainId.MATIC_TESTNET]: ZapVersion.ZapV1,
+    [ChainId.MAINNET]: ZapVersion.ZapV1,
+    [ChainId.ARBITRUM]: ZapVersion.ZapV1,
+    [ChainId.TLOS]: ZapVersion.ZapV1,
+  },
+  [LiquidityDex.Algebra]: {
```

Comment:
I'm feeling concerned about the `LiquidityDex.Algebra` type.

In this case, I would assume it would be `LiquidityDex.Gamma` and there would be a `SwapDex.Algebra`.

Reply:
Now that I think of it. Some documentation explaining the `LiquidityDex` types would be extremely helpful, even if it's a block comment above the interface.

Reply:
Initially, LiquidityDex was for the price getter and that checks algebra pricing (gamma wraps over it so no pricing there). But yeh now we use it for zapVersion too where Gamma is better. So basically neither and both make sense haha